rbc/omniscidb.py

```diff
@@ -314,7 +336,31 @@ def __repr__(self):
         return f'{type(self).__name__}({str(self)!r})'
 
 
-class RemoteOmnisci(RemoteJIT):
+def get_heavydb_version(host='localhost', port=6274, _cache={}):
+    """Acquires the version of heavydb server.
+    """
+    if (host, port) in _cache:
+        return _cache[host, port]
+    thrift_content = '''
+exception TMapDException {
+  1: string error_msg
+}
+service Omnisci {
+  string get_version() throws (1: TMapDException e)
+}
+'''
+    client = ThriftClient(
+        host=host,
+        port=port,
+        multiplexed=False,
+        thrift_content=thrift_content,
+        socket_timeout=60000)
+    version = client(Omnisci=dict(get_version=()))['Omnisci']['get_version']
+    _cache[host, port] = version = parse_version(version)
+    return version
```

Comment:
I guess this is needed because at this point one cannot use `heavyai.get_version()`, right?

Reply:
Correct. In fact, at the point when executing `heavyai.get_version()`, we know what is the brand of heavydb but here we only know what is `host:port` of the heavydb server.

Reply:
In addition, if `dbname` would be the same for different heavydb brands, we would not need this function.